src/test/java/com/training/services/tests/ProductsTest.java

```diff
@@ -61,19 +62,18 @@ public void testShorexTotalHitsValidate() {
      */
     @Test
     public void testShorexProductTypeValidate() {
-        response = new RestEngine().getResponseGet(baseURL, headerMap, queryParam)
-                .as(Response.class);
+        response = new RestEngine().getResponseGet(baseURL, headerMap, queryParam).as(Response.class);
 
-        Assertions.assertThat(response.getStatus()).isEqualTo(200)
-                .as("Json response status is not 200");
+        Assertions.assertThat(response.getStatus()).isEqualTo(200).as("Json response status is not 200");
 
-        for (int i = 0; i < response.getPayload().getProducts().size(); i++) {
+        IntStream.range(0, response.getPayload().getProducts().size()).forEach(index -> {
             softAssert.assertThat(
-                    response.getPayload().getProducts().get(i).getProductType().getProductType()
+                    response.getPayload().getProducts().get(index).getProductType().getProductType()
                             .equalsIgnoreCase("SHOREX"))
-                    .as("Product id " + response.getPayload().getProducts().get(i).getProductID() + " is not SHOREX")
+                    .as("Product id " +
+                            response.getPayload().getProducts().get(index).getProductID() + " is not SHOREX")
                     .isTrue();
-        }
+        });
```

Comment:
Do we really need index in this?
Why cant we use - response.getPayload().getProducts().forEach()?